src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java

```diff
@@ -141,8 +141,12 @@ private void updateEqualSpecimen(List<DigitalSpecimenRecord> currentDigitalSpeci
 
   private Set<DigitalSpecimenRecord> updateExistingDigitalSpecimen(
       List<UpdatedDigitalSpecimenTuple> updatedDigitalSpecimenTuples) {
-    updateHandles(updatedDigitalSpecimenTuples);
-
+    try {
+      updateHandles(updatedDigitalSpecimenTuples);
+    } catch (PidCreationException | PidAuthenticationException e){
+      log.info("Unable to update Handle record. Not proceeding with update");
+      return Set.of();
```

Comment:
Is this correct? So what happens, do we ignore the updates? Might need to dlq those. Also add the specimen or maybe just the physicalIds in the log message so we can which ones failed